.travis.yml

```diff
@@ -35,6 +35,7 @@ install:
   # a low-resource development environment.
   - "echo 'transport.host: 127.0.0.1' >> ./${ES_PATH}/config/elasticsearch.yml"
   - "echo 'http.host: 0.0.0.0' >> ./${ES_PATH}/config/elasticsearch.yml"
+  - ./${ES_PATH}/bin/elasticsearch-plugin install analysis-icu
```

Comment:
This plugin provides the `icu_collation` filter type and the `icu_collation_keyword` field type.
It looks like it's included in AWS Elasticsearch instances by default, but it wasn't on my laptop and obviously it's not here unless we put it here.